app/controllers/user_controller.rb

```diff
@@ -633,4 +633,25 @@ def handle_spam_signup(user_signup)
     end
   end
 
+  def spam_scorer_signup_config
+    {
+      spam_score_threshold: 13,
+      score_mappings: {
+        name_is_all_lowercase?: 1,
+        name_is_one_word?: 1,
+        name_includes_non_alpha_characters?: 1,
+        name_is_garbled?: 1,
+        email_from_suspicious_domain?: 10,
+        email_from_spam_domain?: 13,
+        email_from_spam_tld?: 1,
+        name_is_spam_format?: 10,
+        about_me_includes_currency_symbol?: 1,
```

Comment:
I think 👎 to weighting `about_me`. We don't plan to introduce it on signup at all, so this just communicates that these checks have significance in this case, but in reality they don't do a thing. I'd set all the `about_me*` keys to `0`.

Reply:
p.s. sorry for the initial confusion I added https://github.com/mysociety/alaveteli/pull/4775#issuecomment-411784007

Reply:
fixed (and edited the commit comment to match)